pkg/fake/ec2api.go

```diff
@@ -382,6 +382,42 @@ func (e *EC2API) DescribeLaunchTemplatesWithContext(_ context.Context, input *ec
 	return output, nil
 }
 
+func (e *EC2API) DescribeLaunchTemplatesPagesWithContext(_ context.Context, input *ec2.DescribeLaunchTemplatesInput, callbck func(*ec2.DescribeLaunchTemplatesOutput, bool) bool, _ ...request.Option) error {
```

Comment:
If you look at the way that other calls are defined in the fake api, you can see that we have defined the standard call, in this case we have defined a definition for `DescribeLaunchTemplatesWithContext` elsewhere in the file, and then you can create a `DescribeLaunchTemplatesPagesWithContext` function that calls out to the `DescribeLaunchTemplatesWithContext` directly so you don't have to rewrite the same logic for essentially the same function twice.

Take a look at `DescribeImagesPagesWithContext` for more context